dev-libs/libmelf/libmelf-0.4.0-r1.ebuild

```diff
@@ -11,30 +11,32 @@ SRC_URI="http://www.hick.org/code/skape/${PN}/${P}.tar.gz"
 LICENSE="Artistic"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
-IUSE=""
+IUSE="static-libs"
 
 DEPEND=""
 RDEPEND="${DEPEND}"
 
-src_prepare() {
-	# This patch was gained from the elfsign-0.2.2 release
-	epatch "${FILESDIR}"/${PN}-0.4.1-unfinal-release.patch
-	# Cleanup stuff
-	epatch "${FILESDIR}"/${PN}-0.4.0-r1-gcc-makefile-cleanup.patch
+PATCHES=(
+	"${FILESDIR}"/${PN}-0.4.1-unfinal-release.patch
+	"${FILESDIR}"/${PN}-0.4.0-r1-gcc-makefile-cleanup.patch
+)
+
+src_configure() {
+	default
+	append-flags -fPIC
 }
 
 src_compile() {
-	append-flags -fPIC
-	emake CC="$(tc-getCC)" OPTFLAGS="${CFLAGS}" || die "emake failed"
+	emake CC="$(tc-getCC)" OPTFLAGS="${CFLAGS}" -Wl,soname,libmelf.so
```

Comment:
does this actually work? You need to edit the makefile for this

Reply:
Actually not.  But no clue what in makefile to change ...